docs/machine-learning/how-to-guides/load-data-ml-net.md

```diff
@@ -0,0 +1,146 @@
+
+ML.NET enables you to define data models via classes. For example, given the following input data:
+
+```text
+Size (Sq. ft.), HistoricalPrice1 ($), HistoricalPrice2 ($), HistoricalPrice3 ($), Current Price ($)
+700, 100000, 3000000, 250000, 500000
+1000, 600000, 400000, 650000, 700000
+```
+
+Create a data model that represents the snippet below:
+
+```csharp
+public class HousingData
+{
+    [LoadColumn(0)]
+    public float Size { get; set; }
+ 
+    [LoadColumn(1, 3)]
+    [VectorType(3)]
+    public float[] HistoricalPrices { get; set; }
+
+    [LoadColumn(4)]
+    [ColumnName("Label")]
+    public float CurrentPrice { get; set; }
+}
+```
+
+### Annotating the data model with column attributes
+
+Attributes give ML.NET more information about the data model and the data source.
+
+The [`LoadColumn`](xref:Microsoft.ML.Data.LoadColumnAttribute) attribute specifies your properties' column indices.
+
+> [!IMPORTANT]
+> [`LoadColumn`](xref:Microsoft.ML.Data.LoadColumnAttribute) is only required when loading data from a file.
+
+Load columns as: 
+- Individual columns like `Size` and `CurrentPrices` in the previous `HousingData` class example.
+- Multiple columns at a time in the form of a vector like `HistoricalPrices` in the previous `HousingData` class example.
+
+If you have a vector property, apply the [`VectorType`](xref:Microsoft.ML.Data.VectorTypeAttribute) attribute to the property in your data model. It's important to note that all of the elements in the vector need to be the same type.
+
+By default, ML.NET uses the property names as the column names. However, if you want to change the name of a column to something other than the property name, use the [`ColumnName`](xref:Microsoft.ML.Data.ColumnNameAttribute.#ctor*) attribute. When creating in-memory objects, you still create objects using the property name. However, for data processing and building machine learning models, ML.NET overrides and references the property with the value provided in the [`ColumnName`](xref:Microsoft.ML.Data.ColumnNameAttribute.#ctor*) attribute.
+
+## Load data from a single file
+
+To load data from a file use the [`LoadFromTextFile`](xref:Microsoft.ML.TextLoaderSaverCatalog.LoadFromTextFile*) method along with the data model for the data to be loaded. Since `separatorChar` parameter is tab-delimited by default, change it for your data file as needed. If your file has a header, set the `hasHeader` parameter to `true` to ignore the first line in the file and begin to load data from the second line.
+
+```csharp
+//Create MLContext
+MLContext mlContext = new MLContext();
+
+//Load Data
+IDataView data = mlContext.Data.LoadFromTextFile<HousingData>("my-data-file.csv", separatorChar: ',', hasHeader: true);
+```
+
+## Load data from multiple files
+
+In the event that your data is stored in multiple files, as long as the data model is the same, ML.NET allows you to load data from multiple files that are either in the same directory or multiple directories.
```

Comment:
>ame directory or multiple directories. [](start = 171, length = 38)

I know TLC can load from network shares, so even from other machines, but i have not given this a try with ML.NET